### DISCUSSION TITLE
simultaneous editing

### DISCUSSION THREAD

**Discussion by wdfxl** [outline/outline#3300] [General]
Hi! i enable to use new feature are Simultaneous editing, but i see  SERVER CONNECTION LOST!
![photo_2022-03-29_12-55-26](https://user-images.githubusercontent.com/53470505/160586119-85acaac5-4dee-4329-bbe7-45d822f3e8fe.jpg)

**Comment by almereyda** [comment]
Thanks for giving us notice of this error here. Can you provide some more details about your setup?

Which operating system are you running, how do you deploy Outline and the associated services (Redis, Postgres, S3), and especially which kind of setup are you using to provide TLS termination and routing of web requests?

I suspect this is due to a Websocket connection, which is not properly proxied.

Could you also check the network tab of your browser's developer tools, if there is something that errors out? Ideally you should see HTTP 101 GET requests to the `wss://` endpoints that open a channel for synchronisation messages, in my case visible in the "Response" tab of a selected transmission.
